### DISCUSSION TITLE
Document Template code is not overwriting the TO: field when sending an email.

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1381] [Q&A / Help] [answered]
Hi.
I have a document template set up to email the vet and I have inserted the following code so it will populate the email address for the Vet into the To: field of the email.

<p>{{SUBJECT Vet Work Authority for &lt;&lt;AnimalName&gt;&gt; (&lt;&lt;ShelterCode&gt;&gt;) }} **{{TO &lt;&lt;CurrentVetEmail&gt;&gt;}}** </p>

When the template is selected, it should wipe out the email that is in the TO: field and replace it with the vet email but it is not doing that. The TO: field holds the <CurrentOwnerName> and won't allow it to be overwritten. The Vet email is listed as an option in the dropdown list but it's a bit messy for the user so I wanted to see if there was a change I needed to make so that this TO: field could be blank prior to emailing. Or is there some other solution that can get around it.

Once again. Thanks in advance. Your support is amazing!!!

**Comment by Tetzor** [comment, marked as answer]
Hi,

You can't currently override the "To" field:

https://sheltermanager.com/repo/asm3_help/documents.html#email-tokens-in-document-templates

When we added subject and the other fields, we didn't really see the value in adding "To". In this scenario when a template is always going to be sent to the same recipient, I can see how it might be useful.

I will create a ticket to get this added in a future update.

Kind regards,

Jon
